packages/core/admin/admin/src/content-manager/components/RelationInput/components/tests/Option.test.js

```diff
@@ -44,4 +44,14 @@ describe('Content-Manager || RelationInput || Option', () => {
     expect(getByText('relation 1')).toBeInTheDocument();
     expect(getByTitle('State: Draft')).toBeInTheDocument();
   });
+
+  it('should render custom Option with mainField prop as number type', async () => {
+    const { user, getByRole, getByTitle } = setup({
+      options: [{ id: 1, mainField: 1, publicationState: 'published' }],
+    });
+
+    await user.click(getByRole('combobox'));
+
+    expect(getByTitle('State: Published')).toBeInTheDocument();
```

Comment:
```suggestion
    expect(getByRole('option', { name: 'State: Published' })).toBeInTheDocument();
```

lets tweak this then we're confident we're targeting the option :)

Reply:
Fixed